README.md

```diff
@@ -48,11 +48,13 @@ The following environment variables configure Livebook:
 
   * LIVEBOOK_PASSWORD - sets a password that must be used to access Livebook. Must be at least 12 characters. Defaults to token authentication.
 
-  * LIVEBOOK_PORT - sets the port Livebook runs on. If you want multiple instances to run on the same domain but different ports, you also need to set `LIVEBOOK_SECRET_KEY_BASE`. Defaults to 8080.
+  * LIVEBOOK_PORT - sets the port Livebook runs on. If you want multiple instances to run on the same domain but different ports, you also need to set 'LIVEBOOK_SECRET_KEY_BASE'. Defaults to 8080.
+
+  * LIVEBOOK_SECRET_KEY_BASE - sets a secret key that is used to sign and encrypt the session and other payloads used by Livebook. Must be at least 64 characters long and it can be generated by commands such as: 'openssl rand -base64 48'. Defaults to a random secret on every boot.
 
-  * LIVEBOOK_SECRET_KEY_BASE - sets a secret key that is used to sign and encrypt the session and other payloads used by Livebook. Must be at least 64 characters long and it can be generated by commands such as: `openssl rand -base64 48`. Defaults to a random secret on every boot.
+  * LIVEBOOK_ROOT_PATH - sets the root path to use for file selection.
 
-  * LIVEBOOK_ROOT_PATH - the root path to use for file selection.
+  * LIVEBOOK_IP - sets the ip address to start the web application on. Must be a valid ipv4 or ipv6 address.
```

Comment:
FWIW, I was trying to keep it alphabetical. :)

```suggestion
  * LIVEBOOK_IP - sets the ip address to start the web application on. Must be a valid IPv4 or IPv6 address.
```

Reply:
Good point, I've just sorted options in both places :)